packages/dashboard/src/pages/Wallets/components/FungibleTokenTableRow/index.tsx

```diff
@@ -112,8 +103,8 @@ export const FungibleTokenTableRow = memo<TokenTableRowProps>(({ asset, onSend,
             </TableCell>
             <TableCell className={classes.cell} align="center" variant="body">
                 <Typography>
-                    {new BigNumber(formatBalance(asset.balance, asset.decimals)).gt(pow10(-6))
-                        ? Number.parseFloat(toFixed(formatBalance(asset.balance, asset.decimals) ?? '', 6))
+                    {scale10(asset.balance, asset.decimals).gt(pow10(-6))
```

Comment:
Cause we don't use `formatBalance` for calculation purposes. We can move this case into `formatBalance`.